packages/sanity/src/core/components/commandList/CommandList.tsx

```diff
@@ -567,7 +569,12 @@ export const CommandList = forwardRef<CommandListHandle, CommandListProps>(funct
     inputElement?.setAttribute('aria-expanded', 'true')
     inputElement?.setAttribute('aria-controls', getCommandListChildrenId())
     inputElement?.setAttribute('role', 'combobox')
-  }, [getCommandListChildrenId, inputElement])
+
+    virtualListElement?.setAttribute('aria-autocomplete', 'list')
+    virtualListElement?.setAttribute('aria-expanded', 'true')
+    virtualListElement?.setAttribute('aria-controls', getCommandListChildrenId())
+    virtualListElement?.setAttribute('role', 'combobox')
+  }, [getCommandListChildrenId, inputElement, virtualListElement])
```

Comment:
```suggestion
  }, [getCommandListChildrenId, inputElement])
```

**suggestion**: remove the addition of aria attrs to `virtualListElement`

I don't believe these additions are necessary, as these aria attributes are specific to the input / combobox itself. From what I can tell it should work as expected with these removed – I've noticed that with these enabled, OS X VoiceOver will erroneously report on each item twice when navigating, e.g.
- "Item, Selected, 12/17"
- "Item, Clickable"

When it should just return once, i.e.
- "Item, Selected, 12/17"